src/main/resources/templates/food.html

```diff
@@ -74,16 +76,22 @@ <h2>Food List</h2>
             <tbody>
             <tr th:each="food : ${listFood}">
                 <!--<td th:text="${food.id}">Food ID</td>-->
-                <td><i class="fas fa-drumstick-bite"></i> <span th:text="${food.name}"></span></td>
-                <td th:text="${food.protein}">Protein</td>
-                <td th:text="${food.carbohydrates}">Carbohydrates</td>
-                <td th:text="${food.fat}">Fat</td>
-                <td th:text="${food.energy_kilojoule}">Kilojoule</td>
-                <td th:text="${food.energy_kcal}">Kcal</td>
-                <td class="actionButtons">
-                    <a th:href="@{/updateFood/{id}(id=${food.id})}" class="btn btn-outline-dark mr-2 btn-sm"><i class="fas fa-pen"></i> Edit</a>
-                    <a th:href="@{/deleteFood/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm" th:attr="onclick=|return confirm_delete('${food.name}')|"><i class="fas fa-trash-alt"></i> Delete</a>
-                </td>
+                <th:block th:if="${food.archived == false}">
+                    <td><i class="fas fa-drumstick-bite"></i> <span th:text="${food.name}"></span></td>
+                    <td th:text="${food.protein}">Protein</td>
+                    <td th:text="${food.carbohydrates}">Carbohydrates</td>
+                    <td th:text="${food.fat}">Fat</td>
+                    <td th:text="${food.energy_kilojoule}">Kilojoule</td>
+                    <td th:text="${food.energy_kcal}">Kcal</td>
+                    <td class="actionButtons">
+                        <a th:href="@{/updateFood/{id}(id=${food.id})}" class="btn btn-outline-dark mr-2 btn-sm"><i class="fas fa-pen"></i> Edit</a>
+                        <!--<a th:href="@{/deleteFood/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm" th:attr="onclick=|return confirm_delete('${food.name}')|"><i class="fas fa-trash-alt"></i> Delete</a>-->
+                            <a th:if="${food.fk_user_id == 0}" th:href="@{/archiveFood/true/food/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm"><i class="fas fa-trash-alt"></i> Archive</a>
+                            <a th:if="${food.fk_user_id != 0}" th:href="@{/archiveFood/true/pfood/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm"><i class="fas fa-trash-alt"></i> Archive</a>
```

Comment:
`<i class="fas fa-trash-alt"></i>` should be `<i class="fas fa-archive"></i>` instead